app/views/welcome/index.html.erb

```diff
@@ -62,6 +62,11 @@
     <%# no need to cache this bit either because there's nothing here (done by JS) %>
     <h2 class="interior"><%= t('dashboard.response_locations') %></h2>
     <div class="response_locations"></div>
+    <% if @location_answers_count > @location_answers.size %>
```

Comment:
Hopefully the styling for this is OK. I've got a 13" screen on my MBP, so it's difficult for me to test the styling on the dashboard due to its fancy attempts to recalculate element sizes.